### PR TITLE
Proofreading from Running your server up to Reference > conanfile.py …

### PR DESCRIPTION
**Pull request by dimitrikourk**
…> Requirements

As usual, please let me know if I've messed anything up :)

### CODE REVIEW

File: reference/commands.rst [hunk 1]
```diff
@@ -138,7 +138,7 @@ conan upload
 
 	$ conan upload [--package PACKAGE] [--remote REMOTE] [--all] [--force]
 
-Uploads packages from your local storage to a remote one. If you use ``--force`` variable, it wiil not check the package date, it will be overridden remote with local
+Uploads packages from your local to remote storage. If you use the ``--force`` variable, it wiil not check the package date. It will overridde the remote with the local package.
```
Comment (memsharded) [inline reference/commands.rst]: overridde => override?

File: reference/commands.rst [hunk 2]
```diff
@@ -178,7 +178,7 @@ conan user
 
 	$ conan user [-p PASSWORD] [--remote REMOTE] [name]
 
-Update your cached user [and your password] to avoid it will be requested later, e.g., while you're uploading a package.
-You can have more than one user, and locally manage all your packages from your different accounts
+Update your cached user name [and password] to avoid it being requested later, e.g. while you're uploading a package.
+You can have more than one users, and locally manage all your packages from your different accounts,
```
Comment (memsharded) [inline reference/commands.rst]: I'd say that the "more than one [singular noun]" is not grammatically incorrect

File: server.rst [hunk 3]
```diff
@@ -46,8 +47,8 @@ First, install the conan repository from source and install the requirements:
 
 Server configuration
 --------------------
-Your server configuration lives in ``~/.conan_server/server.conf``, you can change values
-there, prior to launching the server, it is not reloaded if the values are changed, you
+Your server configuration lives in ``~/.conan_server/server.conf``. You can change values
+there, prior to launching the server. Not that the server is not reloaded when the values are changed. You
```
Comment (memsharded) [inline server.rst]: Not? Did you mean "note"?

File: reference/commands.rst (hunk 1 above)
Comment (dimitrikourk) [inline reference/commands.rst]: Absolutely :)

**Comment by memsharded** [comment]
Very minor comments, the PR is a great contribution, let's merge it, thanks very much again!

File: reference/commands.rst (hunk 2 above)
Comment (dimitrikourk) [inline reference/commands.rst]: You're probably right. I just changed that without giving it much thought. Sorry if it's wrong /:)

File: server.rst (hunk 3 above)
Comment (dimitrikourk) [inline server.rst]: Yes :(

**Comment by dimitrikourk** [comment]
:) Thanks for the feedback!

**Comment by memsharded** [comment]
Minor fixes done in: e6557160d4c9627c61d934fb339959db1871240a

**Pull request merged**